From ca70ba180d9f664b3617c4a07618c90c1c3c3acc Mon Sep 17 00:00:00 2001
From: Matt Selsky <matthew.selsky@twosigma.com>
Date: Fri, 9 Dec 2016 00:25:00 -0500
Subject: [PATCH 245/268] Don't name exceptions if you never reference them

pyflakes reported:
pylib/packet.py:934: local variable 'msg' is assigned to but never used
pylib/packet.py:961: local variable 'reason' is assigned to but never used
---
 pylib/packet.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pylib/packet.py b/pylib/packet.py
index ba4331b..142ea69 100644
--- a/pylib/packet.py
+++ b/pylib/packet.py
@@ -931,7 +931,7 @@ class ControlSession:
                 warn("At %s, select with timeout %d begins\n" % (time.asctime(), tvo))
             try:
                 (rd, _, _) = select.select([self.sock], [], [], tvo)
-            except select.error as msg:
+            except select.error:
                 raise ControlException(SERR_SELECT)
             if self.debug > 4:
                 warn("At %s, select with timeout %d ends\n" % (time.asctime(), tvo))
@@ -958,7 +958,7 @@ class ControlSession:
             rpkt = ControlPacket(self)
             try:
                 rpkt.analyze(rawdata)
-            except struct.error as reason:
+            except struct.error:
                 raise ControlException(SERR_UNSPEC)
 
             if rpkt.version() > ntp.magic.NTP_VERSION or rpkt.version() < ntp.magic.NTP_OLDVERSION:
-- 
2.7.4

